### PR TITLE
LibJS: Fix do..while parsing by consuming parentheses explicitly

### PR DESCRIPTION
**Pull request by lawl-dev**
Hi,

this is my first commit and contribute in c++.
Hope it helps :)

kind regards,
Maxim

### CODE REVIEW

**Comment by sunverwerth** [comment]
Good catch.

File: Libraries/LibJS/Parser.cpp [hunk 1]
```diff
@@ -726,9 +726,17 @@ NonnullRefPtr<TryStatement> Parser::parse_try_statement()
...
+
     auto body = parse_statement();
+
     consume(TokenType::While);
+    consume(TokenType::ParenOpen);
+
     auto test = parse_expression(0);
+
+    consume(TokenType::ParenClose);
+    consume(TokenType::Semicolon);
```
Comment (alimpfard) [inline Libraries/LibJS/Parser.cpp]: Aren't we consuming these inside `parse_statement`?

**Comment by lawl-dev** [comment]
I have to squash my commits. I'm on it.

**Comment by awesomekling** [comment]
Cool, thanks for fixing this :)

**Pull request merged**